src/docs/architecture.mdx

```diff
@@ -7,8 +7,8 @@ title: Architecture
 ```mermaid
 graph TD
 app[Your Application] --> |SDK| lb{{Load Balancer}}
-lb --> relay
-lb --> sentry_web["Sentry (web)"]
+lb --> |"ingest.sentry.example.com"| relay
+lb --> |"sentry.example.com/api/\d+/store/"| sentry_web["Sentry (web)"]
```

Comment:
Counterpart to the above comment: This is what handles the web API and serves static routes, so we can show the root URL here:

```suggestion
lb --> |"sentry.example.com"| sentry_web["Sentry (web)"]
```